provider/clusterrosa/classic/cluster_rosa_classic_state.go

```diff
@@ -76,7 +76,8 @@ type ClusterRosaClassicState struct {
 
 	UpgradeAcksFor types.String `tfsdk:"upgrade_acknowledgements_for"`
 
-	DisableWaitingInDestroy types.Bool  `tfsdk:"disable_waiting_in_destroy"`
-	DestroyTimeout          types.Int64 `tfsdk:"destroy_timeout"`
-	WaitForCreateComplete   types.Bool  `tfsdk:"wait_for_create_complete"`
+	DisableWaitingInDestroy     types.Bool  `tfsdk:"disable_waiting_in_destroy"`
+	DestroyTimeout              types.Int64 `tfsdk:"destroy_timeout"`
+	WaitForCreateComplete       types.Bool  `tfsdk:"wait_for_create_complete"`
+	ClusterWaitTimeoutInMinutes types.Int64 `tfsdk:"cluster_wait_timeout_in_minutes"`
```

Comment:
You called it default in the resource but using cluster here in the state, will it work as expected?